tests/publiccloud/sles4sap.pm

```diff
@@ -208,10 +208,11 @@ sub run {
                 # Do the HANA "magic" if needed
                 if ($cluster_type eq 'hana') {
                     my $remoteHost = $hostname;
+                    my $master_resource_type = get_var('USE_SAP_HANA_SR_ANGI') ? "mst" : "msl";
```

Comment:
I can see this file only used in schedule `schedule/publiccloud/ha_sap.yml` 

It this schedule or test module still used anywhere? Is there a way to run a VR for this change? Is this test code related to qe-sap-deployment?

Reply:
DUNNO. It's another hard coded resource name and now it will work much cleaner